src/workloads/sharding/MultiShardTransactions.yml

```diff
@@ -74,6 +74,30 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+
+- Name: QuiesceDatabase0
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database0
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: QuiesceDatabase1
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
```

Comment:
Should `QuiesceDatabase1` be done in a phase after `QuiesceDatabase0` like how is done in `src/workloads/sharding/MultiShardTransactionsWithManyNamespaces.yml`?